plugins/GeoIp2/GeoIP2AutoUpdater.php

```diff
@@ -464,7 +464,7 @@ protected static function checkGeoIPUpdateUrl($url)
         $host = $parsedUrl['host'] ?? '';
 
         if (empty($schema) || empty($host) || !in_array(Common::mb_strtolower($schema), ['http', 'https'])) {
-            throw new Exception(Piwik::translate('GeoIp2_MalFormedUpdateUrl', '<i>'.$url.'</i>'));
```

Comment:
Looks good, but I'm wondering if we should use `Common::sanitizeString()` for consistency?

Reply:
Makes sense. changed it to `Common::sanitizeInputValue()` as the other one has private access only